core/email_api.php

```diff
@@ -1300,6 +1300,16 @@ function email_send( EmailData $p_email_data ) {
 			break;
 	}
 
+	#apply DKIM settings
+	if( config_get( 'dkim_enable' ) ) {
+		$t_mail->DKIM_domain = config_get( 'email_dkim_domain' );
+		$t_mail->DKIM_private = config_get( 'email_dkim_private_key_file_path' );
```

Comment:
I suggest only setting the file path if the key string is not set.

Reply:
Why adding extra code when this is already handled by PHPMailer. We have specified in the documentation and comments the order the settings are applied in.